package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "go build -o build/ethier ethier/*.go",
     "generate": "PATH=\"$(readlink -f build):${PATH}\" go generate ./...",
+    "solhint": "npx solhint 'contracts/**/*.sol' && npx solhint 'tests/**/*.sol' && npx solhint 'factories/**/*.sol'",
```

Comment:
We could also replace this with `npx solhint '**/*.sol'` and add a `.solhintignore` file to exclude `node_modules`.
This way we don't have to worry about adding new folders if we add some in the future. Sorry for not thinking about this earlier.